app/src/main/res/values/strings.xml

```diff
@@ -109,31 +109,31 @@
     <string name="apps_filter_sharedid_label">Shared User ID</string>
-    <string name="apps_filter_multipleprofiles_label">Multiple profiles</string>
+    <string name="apps_filter_multipleprofiles_label">Apps with Clones</string>
     <string name="support_debuglog_label">Debug log</string>
     <string name="support_debuglog_desc">Record everything the app is doing into a text file that you can share.</string>
     <string name="updated_at_x">Last updated: %s</string>
     <string name="installed_at_x">First installed: %s</string>
     <string name="apps_sort_install_source_label">Install source</string>
     <string name="permissions_action_none_msg">No action available for this permission</string>
-    <string name="apps_filter_profile_secondary_label">Secondary profile</string>
+    <string name="apps_filter_profile_secondary_label">Apps in Managed Profile(s)</string>
     <string name="apps_details_description_restrictions_caveat_description">Due to Android restrictions, it is not possible to determine which permissions have been granted to apps installed in Managed User Profiles (i.e. Work Profile and Multi-Users), from the Main Profile. Permission status can only be shown for apps in the Active Profile (i.e. current profile). To view the permission status in other profiles, install this app in those profiles.</string>
     <string name="apps_details_description_secondary_description">"%d permissions requested."</string>
-    <string name="apps_details_description_primary_description">"%1$d of %2$d permissions granted."</string>
+    <string name="apps_details_description_primary_description">"%1$d of %2$d (%s) permissions granted."</string>
     <string name="api_target_level_x">Target version: %s</string>
     <string name="api_minimum_level_x">Minimum version: %s</string>
     <string name="api_build_level_x">Compile version: %s</string>
     <string name="permission_reboot_label">Reboot device</string>
     <string name="permission_reboot_description">Allows the app to reboot the device. Only available to system apps.</string>
     <string name="permissions_sort_apps_relevance_label">Relevance</string>
-    <string name="apps_filter_profile_active_label">Active profile</string>
+    <string name="apps_filter_profile_active_label">Apps in Active profile</string>
     <string name="overview_page_label">Overview</string>
     <plurals name="generic_x_items_label">
         <item quantity="one">%s item</item>
         <item quantity="other">%s items</item>
     </plurals>
-    <string name="permissions_details_count_user_apps">"User Apps: Granted to %1$d out of %2$d."</string>
-    <string name="permissions_details_count_system_apps">"System Apps: Granted to %1$d out of %2$d."</string>
-    <string name="permissions_details_count_total_apps">"Total: Granted to %s."</string>
-
+    <string name="permissions_details_count_user_apps">"User Apps: Granted to %s (%1$d out of %2$d) of User apps."</string>
```

Comment:
You can't just add extra placerholders without changing the code, it will crash because it tries to place 2 variables in 3 placeholders.

Reply:
This will crash

https://github.com/d4rken-org/permission-pilot/blob/ee4a87666e4bbf41dee00ccacf07cb87c634709c/app/src/main/java/eu/darken/myperm/permissions/ui/details/items/PermissionOverviewVH.kt#L60-L64

Reply:
Okay. I didn't know the code that runs in the background.

How did this commit pass the GitHub tests?

Anyways, can you please fix them?

Reply:
Please check line 122 also. I made a change there too.

Reply:
> How did this commit pass the GitHub tests?

Someone has to write the tests that test this :stuck_out_tongue:  
Currently we just check for build errors, and run a few tests that I copied from other projects of mine.

Reply:
So it is a check that the developer himself writes (in this case you) for each of his projects? Not some standard tests provided by GitHub?

Reply:
> So it is a check that the developer himself writes (in this case you) for each of his projects? Not some standard tests provided by GitHub?

Correct